jobserver/views/job_requests.py

```diff
@@ -64,7 +65,7 @@ def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
         context["project_definition"] = mark_safe(
             render_definition(
-                self.object.project_definition,
+                escape(self.object.project_definition),
```

Comment:
Should the escaping be done inside the `render_definition` function as that's the thing which is adding the additional HTML?

Reply:
I don't have a strong opinion here.  I was trying to keep `render_definition` "pure", in that it didn't deal with sanitisation/escaping, but transformed a string -> HTML.  However I could see the argument that expecting a caller to escape input is asking for trouble.

Do you feel strongly about moving it in?

Reply:
Don't feel strongly, no.